Shared/DataTransferObjects/AttendanceForCreationDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceForCreationDto(
+    Guid EmployeeId,
+    DateTime CheckInTime,
+    string Status,
```

Comment:
Consider replacing the 'Status' string with an enum to ensure only valid attendance statuses (e.g., Present, Absent, Late) are allowed. This improves type safety and prevents invalid values from being passed into the system.

```suggestion
AttendanceStatus Status
```